ui/explore/src/components/ExploreManager/constants.ts

```diff
@@ -11,5 +11,5 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export const PANEL_PREVIEW_HEIGHT = 300;
+export const PANEL_PREVIEW_HEIGHT = 700;
```

Comment:
This does not apply to graph ? isn't it weird in graph ?

Reply:
I think it's okay: 
![image](https://github.com/perses/perses/assets/5657041/5bb557ef-a4fb-4c03-a3ff-31117b982672)